lms/djangoapps/email_marketing/migrations/0003_auto_20160714_0952.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0002_auto_20160623_1656'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_abandoned_cart_delay',
+        ),
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_abandoned_cart_template',
+        ),
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_purchase_template',
+        ),
+        migrations.RemoveField(
```

Comment:
I don't have a good solution, yet. For the moment, let's just leave the fields and mark them as deprecated.